index.html

```diff
@@ -60,6 +61,188 @@ <h4 class="title">I'am a Software Developer</h4>
+          <h3 class="project-card-heading">
+            Professional Art Printing Data
+          </h3>
+          <p class="project-card-description">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+          </p>
+          <ul class="skills">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button class="see-project">
+            See Project
+          </button>
+        </div>
+        <div class="project-card inactive">
+          <img src="./images/background2.png" alt="Professional Art Printing Data">
+          <h3 class="project-card-heading">
+            Professional Art Printing Data
+          </h3>
+          <p class="project-card-description">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+          </p>
+          <ul class="skills">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button class="see-project">
+            See Project
+          </button>
+        </div>
+        <div class="project-card inactive">
+          <img src="./images/background2.png" alt="Professional Art Printing Data">
+          <h3 class="project-card-heading">
+            Professional Art Printing Data
+          </h3>
+          <p class="project-card-description">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+          </p>
+          <ul class="skills">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button class="see-project">
+            See Project
+          </button>
+        </div>
+        <div class="project-card inactive">
+          <img src="./images/background2.png" alt="Professional Art Printing Data">
+          <h3 class="project-card-heading">
+            Professional Art Printing Data
+          </h3>
+          <p class="project-card-description">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+          </p>
+          <ul class="skills">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button class="see-project">
+            See Project
+          </button>
+        </div>
+      </div>
+    </section>
+    <!-- about me section -->
+    <section id="about-me">
+      <div class="about-me-intro">
+        <h2 id="about-me-title">About me</h2>
+        <p id="about-me-description">
+          Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
```

Comment:
<img width="715" alt="1" src="https://user-images.githubusercontent.com/77843050/219604911-8df89787-c268-4a0f-99d7-2c8572634b21.png">

- [ ] Hey @Bennyjoez, you have done great jobs on this work, most of the requirement are respected and that makes me happy though, but I have a few issue to raise regarding the design, take a look at the image above, your paragraph text is not sharing the same font-weight as the original design, please make sure you update that
- [ ] and else the image background should a little bit move to the bottom as shown on the original design, kindly make sure fix this too.